macros/src/test/scala/gestalt/macros/DefMacroTest.scala

```diff
@@ -11,6 +11,15 @@ class DefMacroTest extends TestSuite {
 
     assert(plusObject.poly(3, 5) == 8)
     assert(plusObject.poly("3", 5) == 8)
+
+    val five = 5
+    assert(plusObject.varargs(1, 1 + 1, five) == 8)
+    assert(plusObject.varargs(1, 2, 5) == 8)
+//    assert(plusObject.varargs(Seq(1, 2, 5):_*) == 8)
```

Comment:
This is commented out for now because of a somewhat unrelated bug. Details soon.